.babelrc

```diff
@@ -1,15 +1,34 @@
 {
-  "presets": ["next/babel"], // https://goo.gl/aqbWJY
+  "presets": [[
+    "next/babel",
+    {
+      "preset-env": {
+        "useBuiltIns": "entry",
+        "corejs": 3,
```

Comment:
Newer than the default. Default would be 2, also this line is necessary to get rid of a warning.